### PR TITLE
Fix autoreset timer renewal for unchanged physical light states

### LINKED ISSUE #1530: Autoreset timer keeps restarting with detect_non_ha_changes enabled
Reproduced in the integration tests on HA 2026.4.3 while checking #1508, with the fix from #1525 applied.

1. Set `detect_non_ha_changes: true`, `autoreset_control_seconds: 7200`, and `interval: 90`.
2. Manually change brightness or color temperature away from the adaptive target.
3. Run an adaptation interval after 90 seconds.

The remaining timeout returns to 7200 instead of falling to 7110. This happens with both `pause_all` and `pause_changed`, with interception on or off.

The untracked-change check compares the unchanged manual value against an old adaptive target in `last_service_data`. Each poll treats the same difference as a new manual change and restarts the timer. A fix must still detect later physical dimmer changes; ignoring already-manual attributes would lose those.

This is separate from #1525's service-event fix. The option is disabled by default.

### PR DESCRIPTION
**Pull request by basnijholt**
With `detect_non_ha_changes` enabled, repeated polls of an unchanged manual value kept restarting the autoreset timer. Track the last reported manual change per axis, including the result of tracked HA commands. Later physical changes still renew the timer, even when devices reuse a context ID. Adaptive Lighting writes clear only the baseline for the axis they change, so `apply` does not cause false renewals.

Closes #1530.

Validation: reproduced the physical-poll, tracked-command, and forced-apply cases before fixing them. All 360 tests pass on HA 2026.9.1, 24 focused cases pass on HA 2025.9.4, and repository hooks pass. Coverage includes both takeover modes, interception on/off, brightness/color, cumulative small changes, and reused contexts. Also fixed two existing test helpers that accidentally sent brightness zero when the adaptive value was 155; all five affected cases pass with that value injected.

### CODE REVIEW

File: custom_components/adaptive_lighting/switch.py [hunk 1]
```diff
@@ -2805,13 +2830,39 @@ async def significant_change(
...
             light=light,
             context=context,
         )
+        manual_control = self.get_manual_control_attributes(light)
+        manual_control_state_attributes = (
+            self.last_manual_control_state_attributes.get(
+                light,
+                LightControlAttributes.NONE,
+            )
+            & manual_control
```
[the remark this answers is not in the document]
Comment (basnijholt) [reply custom_components/adaptive_lighting/switch.py]: Fixed in 7c143e4. The post-command state now seeds each changed axis once. Tests cover the duplicate renewal and later physical changes that reuse the same context ID.

**Comment by basnijholt** [comment]
@greptileai review the updated baseline handling in 7c143e4.
